tests/sourcerer/storage_provider/test_services.py

```
@@ -607,13 +619,300 @@ def test_get_file_size_error(self):
+        with patch.object(self.service, 'get_containers_client', return_value=self.mock_blob_service_client):
+            # Act - Test listing containers (empty path)
+            result1 = self.service.list_storage_items(self.test_storage, "", "")
+
+            # Assert
+            self.mock_blob_service_client.list_containers.assert_called_once()
+            self.assertEqual(len(result1.folders), 2)
+            self.assertIn("container1", [i.key for i in result1.folders])
+            self.assertIn("container2", [i.key for i in result1.folders])
+
+            # Act - Test listing blobs in a container
+            result2 = self.service.list_storage_items(self.test_storage, self.test_container, "")
```

Ruff (0.8.2)

Local variable `result2` is assigned to but never used

Remove assignment to unused variable `result2`

(F841)